shaders/src/material_inputs.fs

```diff
@@ -88,6 +88,15 @@ struct MaterialInputs {
 #endif
 #endif
 #endif
+
+#if defined(MATERIAL_HAS_SPECULAR_FACTOR)
+    float specularFactor;
```

Comment:
See comments above about naming for those two inputs

Reply:
Will work on next.